docs/preview/03-Features/correlation.md

```diff
@@ -93,7 +93,9 @@ app.UseHttpCorrelation();
 app.UseRouting();
 ```
 
-Note: because the correlation is based on <span>ASP.NET</span> Core middleware, it's recommended to place it before the `.UseRouting` call.
+> Note: because the correlation is based on <span>ASP.NET</span> Core middleware, it's recommended to place it before the `.UseRouting` call.
```

Comment:
In this case, I would say recommended bc it doesn't require routing but it is wise to make sure that it is done as soon as possible so that the later calls are correlated.